src/custom-element.js

```diff
@@ -444,9 +444,6 @@ function createBaseCustomElementClass(win) {
       this.layout_ = Layout.NODISPLAY;
 
       /** @private {number} */
-      this.layoutWidth_ = -1;
```

Comment:
What's going to happen to this var? I still see it used throughout in this class.